cmd/infracost/output.go

```diff
@@ -80,10 +80,20 @@ func outputCmd(cfg *config.Config) *cobra.Command {
 
 			format, _ := cmd.Flags().GetString("format")
 
+			fields := []string{"monthly_quantity", "unit", "monthly_cost"}
+			if cmd.Flags().Changed("fields") {
+				if c, _ := cmd.Flags().GetStringSlice("fields"); len(c) == 0 {
+					ui.PrintWarning("'--fields' flag is empty, set a defaults")
```

Comment:
```suggestion
					ui.PrintWarning("'--fields' flag is empty, using defaults")
```
If we want we could even show the defaults here using something like
```suggestion
					ui.PrintWarningf("'--fields' flag is empty, using defaults: %s", cmd.Flag("fields").DefValue)
```
What do you think?

Reply:
Sure, it looks useful, will add it